MagickCore/resize.c

```diff
@@ -414,16 +414,30 @@ static double Lagrange(const double x,const ResizeFilter *resize_filter)
   return(value);
 }
 
-static double MagicKernelSharp(const double x,
-  const ResizeFilter *magick_unused(resize_filter))
+static double MagicKernelSharp(const double x,const ResizeFilter *resize_filter)
 {
-  magick_unreferenced(resize_filter);
-
   /*
-    Magic Kernel with Sharp2021 filter
+    Magic Kernel Sharp
 
     See: Solving the mystery of Magic Kernel Sharp (https://johncostella.com/magic/mks.pdf)
   */
+  if (resize_filter->support <= 2.5)
```

Comment:
Maybe this should be a separate method instead of a check inside this method?

Reply:
Done